crates/biome_analyze/src/rule.rs

```diff
@@ -373,6 +373,10 @@ pub enum RuleDomain {
     Solid,
     /// Next.js framework rules
     Next,
+    /// Vue.js 2 & 3 framework rules
+    Vue,
+    /// Vue.js 3 framework rules
+    Vue3,
```

Comment:
Since this is an empty domain (for now), you may want to update the codegen of the website to **not** generate code for the domain if there are no rules that belong to it. The code is in the `next` branch

Reply:
Will do